src/ui/ResultLink/ResultLink.ts

```diff
@@ -471,6 +471,14 @@ export class ResultLink extends Component {
     true
   );
 
+  private filterProtocol(uri: string) {
+    if (!/^https?:\/\//.test(uri)) {
```

Comment:
Hum. Didn't we say we wanted to blacklist `javascript:` only ? 

Whitelisting to https only won't work... Many many client still index http websites, they might want to link to `file://`, they might want to link to other protocol.